Backward incompatible change with Tranquility

@gianm @cheddar 

https://github.com/druid-io/druid/pull/4060 is causing a backward incompatible change which is breaking existing Tranquility, here is the error
`2017-03-22 14:02:06.527 y.s.a.s.DruidBeamSink TranquilitySink_[5 5] [ERROR] TranquilitySink - Problem preparing DruidBeam bolt in worker 5: java.lang.RuntimeException: org.codehaus.jackson.map.exc.UnrecognizedPropertyException: Unrecognized field "enabled" (Class org.apache.shaded.curator.x.discovery.ServiceInstance), not marked as ignorable
 at [Source: [B@1913fa09; line: 1, column: 234] (through reference chain: org.apache.shaded.curator.x.discovery.ServiceInstance["enabled"])``

--- a/pom.xml
+++ b/pom.xml
@@ -58,7 +58,7 @@
     </scm>
 
     <properties>
-        <apache.curator.version>2.12.0</apache.curator.version>
+        <apache.curator.version>2.11.0</apache.curator.version>
         <avatica.version>1.9.0</avatica.version>
         <calcite.version>1.11.0</calcite.version>
         <guava.version>16.0.1</guava.version>

